### DISCUSSION TITLE
blog/2024/08/20/%E9%BB%91%E5%90%97%E5%96%BD-%E7%A5%9E%E8%AF%9D/

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#17] [General]
# blog/2024/08/20/%E9%BB%91%E5%90%97%E5%96%BD-%E7%A5%9E%E8%AF%9D/

黑吗喽玩上了吗！？大伙儿

https://996.ninja/blog/2024/08/20/%E9%BB%91%E5%90%97%E5%96%BD-%E7%A5%9E%E8%AF%9D/

**Comment by healthy2bro** [comment]
怎么没有1999年央视动画部制作的经典动画《西游记》，共52集，讲述了唐僧师徒四人取经的冒险故事。

**Reply by MingGH** [reply]
你说的有道理🥸，晚上回去更新下

**Comment by MingGH** [comment]
还看到个好玩的


你们说话啊！现在就黑神话了？
你们说话啊！现在就黑神话了？
你们说话啊！现在就黑神话了？
你们说话啊！现在就黑神话了？
你们说话啊！现在就黑神话了？
你们说话啊！现在就黑神话了？
留我一个人上班？😭 留我一个人上班？😭 留我一个人上班？😭 留我一个人上班？😭 留我一个人上班？😭 留我一个人上班？😭 留我一个人上班？😭 留我一个人上班？😭 留我一个人上班？😭 留我一个人上班？😭 留我一个人上班？😭 留我一个人上班？😭 
捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡捅死你😡
